dashboard/app/views/levels/_external.html.haml

```diff
@@ -6,12 +6,11 @@
 :javascript
```

Comment:
Not part of your change, but should line 5 of this file (which includes external.js) be inside a `content_for :head` block?